File: Data Structures & Algorithms/kth-largest-element-in-an-array/submission-2.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        int[] count = new int[100001];

        for(int num : nums){
            count[num]++;
        }

        int n = count.length;
        for(int i=n-1; i>=0; i--){
            while(count[i] > 0){
                count[i]--;
                k--;

                if(k == 0){
                    return i;
                }
            }
        }
        return -1;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-2.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        seen = [0] * 1001
        nums.sort()
        if len(nums) <= 0:
            return 0

        max_count = 1
        count = 1
        for R in range(len(nums) - 1):
            
            if nums[R] - nums[R + 1] == -1:
                count += 1
            elif nums[R] == nums[R + 1]:
                continue
            else:
                count = 1
            max_count = max(max_count, count)

        return max_count
